index.html

```diff
@@ -0,0 +1,274 @@
+        <div class="card">
+            <img src="https://files.auroville.org/auroville-org/15e601a8-62a1-4db0-b5ef-33be87beb133.jpg" 
+            alt="Integral Yoga, the Process"
+            id="integral-yoga">
+            <div class="container">
+                Integral Yoga, the Process
+            </div>
+        </div>
+    </div>
+    <h1 id="overview">Overview</h1>
+    <div id="overview-content">
+        <div class="flex">
+        <div>1968<p>The Year of Founding</p></div>
+        <div>124<p>Nations Participated</p></div>
+        <div>5000<p>People Attended</p></div>
+        </div>
+        <div class="flex">
+        <div>5km<p>Planned City Diameter</p></div>
+        <div>50000<p>Planned Population</p></div>
+        <div>3300<p>Current Population</p></div>
+        </div>
+    </div>
+    <h1 id="explore">Explore the Inspiring Vision of Auroville</h1>
+    <div class="grid-2">
+        <a href="https://auroville.org/page/core-documents">
+        <div class="cards">
+            <img src="https://files.auroville.org/auroville-org/3332822d-41a5-44f5-bb2b-9f5515502584.jpg"
+            alt="Matrimandir"
+            id="matrimandir">
+            <div class="container-2">
+                <h4>3 Core Documents</h4>
+                <p>The Auroville Charter, A Dream and To Be A True
+                    Aurovillian - these 3 sentimental documents define the
+                    essence of auroville
+                </p>
+            </div>
+        </div>
+        </a>
+        <a href="https://auroville.org/page/vision-of-the-city">
+            <div class="cards">
+                <img src="https://files.auroville.org/auroville-org/a25e6be7-2458-47c4-b2bc-49db82a33d3e.jpg"
+                alt="vision of the city"
+                id="visionb">
+                <div class="container-2">
+                    <h4>Vision Of The City</h4>
+                    <p>Auroville has been conceived as a city in a form of a
+                    galaxy with the Matrimandir at the centre
+                    representing the soul of the city
+                </p>
+                </div>
+            </div>
+        </a>
+        <a href="https://auroville.org/page/the-founders">
+            <div class="cards">
+                <img  src="https://files.auroville.org/auroville-org/fdbbef79-bbfd-4d04-b5d8-3c7f2a5329ed.jpg"
+                alt="mother"
+                id="mother">
+                <div class="container-2">
+                    <h4>The Founders</h4>
+                    <p>Auroville was founded by The Mother (Mirra Alfassa)
+                       who was the spiritual collabrator of the Indian Sage
+                       Sri Aurobindo
+                    </p>
+                </div>
+            </div>
+        </a>
+        <a href="https://auroville.org/page/integral-yoga">
+            <div class="cards">
+                <img src="https://files.auroville.org/auroville-org/a21d0974-55b2-4ab3-ab62-ced4db9c90be.jpg"
```

Comment:
Use relative file paths for all of the images which is in this page. Read the below documentation on file paths.
https://www.w3schools.com/html/html_filepaths.asp